WalletWasabi/JsonConverters/KeyJsonConverter.cs

```diff
@@ -17,7 +17,18 @@ public override bool CanConvert(Type objectType)
 		/// <inheritdoc />
 		public override object ReadJson(JsonReader reader, Type objectType, object existingValue, JsonSerializer serializer)
 		{
-			return Key.Parse(((string)reader.Value).Trim());
+			var serializedKey = ((string)reader.Value).Trim();
+			foreach (var network in Network.GetNetworks())
+			{
+				try
+				{
+					return Key.Parse(serializedKey, network);
+				}
+				catch (FormatException)
+				{
+				}
```

Comment:
YahihebFactor fails

Reply:
Internally these codes have an internal logic and could return true/false with TryParse. Exceptions are slow, but in this case, there is no other option.

Reply:
I don't understand your comment @nopara73

Reply:
empty catch block -> @yahiheb -> CodeFactor -> YahihebFactor

Reply:
In your opinion what should I do in case the base58 serialized key is not for the network?

Reply:
IMO we should specify which networks we are waiting for here.

try {return Key.Parse(serializedKey, Network.Main);}
catch { try {return Key.Parse(serializedKey, Network.TestNet);}}
...
catch {return null;}